app/src/renderer/components/staking/PageBond.vue

```diff
@@ -125,7 +126,6 @@ tm-page.page-bond(:title="`Bond ${denom}`")
 import { between, numeric, required } from "vuelidate/lib/validators"
 import { mapGetters } from "vuex"
 import num from "scripts/num"
-import interact from "interactjs"
```

Comment:
do we still need the interactjs dependency?

Reply:
good call! i'll remove